core/src/test/java/org/aerogear/mobile/core/integration/MetricsServiceIntegrationTest.java

```diff
@@ -29,8 +38,22 @@
 
     @Before
     public void setUp() {
-        MobileCore.init(RuntimeEnvironment.application);
-        metricsService = new MetricsService("https://demo1623828.mockable.io/metrics");
+        Application context = RuntimeEnvironment.application;
+        MobileCore.init(context);
+
+        // read the app metrics endpoint from mobile-services.json
+        String metricsEndpoint = null;
+        try (InputStream configStream = context.getAssets().open("mobile-services.json")) {
+            MobileCoreConfiguration jsonConfig = new MobileCoreJsonParser(configStream).parse();
+            Map<String, ServiceConfiguration> configsPerId = jsonConfig.getServicesConfigPerId();
+            ServiceConfiguration metricsConfig = configsPerId.get("metrics-myapp-android");
+            metricsEndpoint = metricsConfig.getUrl();
+        } catch (JSONException | IOException exception) {
+            System.err.println(exception);
+            fail(exception.getMessage());
+        }
+
+        metricsService = new MetricsService(metricsEndpoint);
```

Comment:
The `MobileCore.init()` automatically create a MetricsService instance so you can replace all of this by `MobileCore.getInstance().getMetricsService()`